### DISCUSSION TITLE
[mini.comment] support for comment in next line

### DISCUSSION THREAD

**Discussion by kskarthik** [nvim-mini/mini.nvim#341] [Ideas]
Hi, Thank your for the great plugin! 

I am coming from https://github.com/numToStr/Comment.nvim which has such feature.

It would be really nice to have here too!

config option can be `comment_next_line="gco"`

**Comment by echasnovski** [comment]
Thanks for suggestion!

This seems like a relatively niche feature which can mostly be covered by mapping. There are at least two options here:
- `vim.keymap.set('n', 'gco', 'jgcck', { remap = true })` - which is basically saying "Go down - toggle line comment - go up". This moves cursor to the first non-blank character, though (which might be acceptable).
- `vim.keymap.set('n', 'gco', "<Cmd>lua local l = vim.fn.line('.') + 1; MiniComment.toggle_lines(l, l)<CR>")` - which is basically saying "Toggle comment on the next line only".

Both of them won't properly work on the last line and won't support dot-repeat, but still are fairly good solutions.

**Reply by kskarthik** [reply]
great! let me try them out :)
